README.md

```diff
@@ -265,10 +265,10 @@ login.gov team for credentials and other values.
 
 ### Managing translation files
 
-To help us handle extra newlines and make sure we wrap lines consistently, we have a script called `./script/normalize-yaml` that helps format YAML consistently. After importing translations (or making changes to the *.yml files with strings, run this for the IDP app:
+To help us handle extra newlines and make sure we wrap lines consistently, we have a script called `./scripts/normalize-yaml` that helps format YAML consistently. After importing translations (or making changes to the *.yml files with strings, run this for the IDP app:
```

Comment:
Should we replace the reference to `scripts/normalize-yaml` with `make normalize_yaml` since that is the recommended command to run?

Reply:
the diff doesn't tell the whole story :) immediately below that paragraph it says 
```
$ make normalize_yaml
```

Reply:
I know. I added that to the README a while back because people didn't know which command to run or whether or not it needed arguments. Looking back, I should have updated the paragraph above as well, so I thought since you're in there now, might as well do it.